webapp/tests/test_render.py

```diff
@@ -132,3 +158,77 @@ def test_correct_timezone(self):
         # all the from/until/tz combinations lead to the same window
         expected = [[12, 1393398060], [12, 1393401660]]
         self.assertEqual(data, expected)
+
+    def test_template_numeric_variables(self):
+        url = reverse('graphite.render.views.renderView')
+        response = self.client.get(url, {
+                 'target': 'template(constantLine($1),12)',
+                 'format': 'json',
+                 'from': '07:01_20140226',
+                 'until': '08:01_20140226',
+        })
+        data = json.loads(response.content)[0]['datapoints']
+        # all the from/until/tz combinations lead to the same window
+        expected = [[12, 1393398060], [12, 1393401660]]
+        self.assertEqual(data, expected)
+
+        url = reverse('graphite.render.views.renderView')
+        response = self.client.get(url, {
+                 'target': 'template(constantLine($num),num=12)',
+                 'format': 'json',
+                 'from': '07:01_20140226',
+                 'until': '08:01_20140226',
+        })
+        data = json.loads(response.content)[0]['datapoints']
+        # all the from/until/tz combinations lead to the same window
+        expected = [[12, 1393398060], [12, 1393401660]]
+        self.assertEqual(data, expected)
```

Comment:
Added the new tests and made sure numbers passed via query parameters are parsed correctly.